### PR TITLE
Patch panel v2

### PR DESCRIPTION
**Pull request by carlos-soncco**
The last version of patch_panel

### CODE REVIEW

File: Detectors/ITSMFT/MFT/base/include/MFTBase/PatchPanel.h [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+{
+
+ public:
+  PatchPanel();
+  ~PatchPanel() = default;
+
+  TGeoVolumeAssembly* createPatchPanel();
+
+  //protected:
+  //TGeoVolumeAssembly* mPatchPanel;
```
Comment (sawenzel) [inline Detectors/ITSMFT/MFT/base/include/MFTBase/PatchPanel.h]: there are multiple commented lines here and below that appear to be deletable. Could you still remove them?

**Comment by sawenzel** [comment]
@bovulpes : Could you approve this PR?

**Comment by sawenzel** [comment]
Can someone from MFT @bovulpes @frmanso approve this?

**Comment by sawenzel** [comment]
Reminding call for approval from @bovulpes @frmanso. Is there any news on this PR?

**Comment by frmanso** [comment]
Ok, I will check this pull request

**Comment by frmanso** [comment]
Carlos, it's missing the code where you really create the patch panel... I would include it in GeometryBuilder file.

**Comment by frmanso** [comment]
Still need for work before to validate this pull request. Sorry for the delay.

**Comment by sawenzel** [comment]
Ok. I will close this PR in order not to take ressources in the CI. Please take your time to rework and re-open the PR when you think it's ready.

**Pull request closed without merging**